softwerkskammer/moveUsersFromEzmlmToGroups.js

```diff
@@ -0,0 +1,72 @@
+/*eslint no-process-exit: 0 */
+/* eslint no-console: 0 */
+'use strict';
+
+const async = require('async');
+
+const conf = require('./configure'); // initializing parameters
+const beans = require('simple-configure').get('beans');
+const memberstore = beans.get('memberstore');
+const groupstore = beans.get('groupstore');
+//Just checking if remote has been configured
+const listAdapter = conf.get('fullyQualifiedHomeDir') ? beans.get('ezmlmAdapter') : beans.get('fakeListAdapter');
+
+const really = process.argv[2];
+if (!really || really !== 'really') {
+  console.log('This script migrates listusers to groups.');
+  console.log('If you want to run this script, append "really" to the command line.');
+  process.exit();
+}
+
+memberstore.allMembers((err, members) => {
+  if (err) {
+    console.log(err);
+    process.exit();
+  }
+
+  function listsWithEmails(callback) {
+    function createListUsersObject(list, cb) {
+      listAdapter.getUsersOfList(list, (errX, users) => {
+        cb(errX, {listname: list, users});
+      });
```

Comment:
Die geschweiften Klammern an dem Callback sind doch überflüssig, oder? Dann kann das in eine Zeile vielleicht...